File: app/src/main/java/com/proxy/shadowsocksr/impl/obfs/SSObfs.java
package com.proxy.shadowsocksr.impl.obfs;

public class SSObfs extends AbsObfs
{
    public SSObfs(String usrParamStr, String rmtIP, int rmtPort, int tcpMss)
    {
        super(usrParamStr, rmtIP,rmtPort, tcpMss);
    }

    @Override public byte[] afterEncrypt(byte[] data)
    {
        return data;
    }

    @Override public byte[] beforeDecrypt(byte[] data, boolean needsendback)
    {
        return data;
    }
}

File: jvmVer/src/main/java/com/proxy/shadowsocksr/impl/plugin/obfs/SSObfs.kt
package com.proxy.shadowsocksr.impl.plugin.obfs

import java.util.*

class SSObfs(usrParamStr: String, rmtIP: String, rmtPort: Int, tcpMss: Int, shareParam: HashMap<String, Any>) : AbsObfs(
        usrParamStr, rmtIP, rmtPort, tcpMss, shareParam)
{
    @Throws(Exception::class)
    override fun afterEncrypt(data: ByteArray): ByteArray
    {
        return data
    }

    @Throws(Exception::class)
    override fun beforeDecrypt(data: ByteArray, needsendback: Boolean): ByteArray
    {
        return data
    }
}
